### DISCUSSION TITLE
Change Metadata in Detailarea

### DISCUSSION THREAD

**Discussion by Ulfenknulfen** [LesFerch/WinSetView#117] [Ideas]
Hi!

In the past days of win 10/11 it was possible to change Metadata in the 
the Detailarea directly. No it is not longer possible.
Is there a known Registry key, or fix that enable the funktion?

best regards
Paddy

**Comment by LesFerch** [comment]
Currently, there is no known method to bring back direct editing of file properties in the Details pane of the Windows 11 Explorer. The only way to get that capability is to use the Windows 10 Explorer in Windows 11. See my [SwitchExplorer](https://lesferch.github.io/SwitchExplorer/) and [OldExplorer](https://lesferch.github.io/OldExplorer/) tools for more details.

**Reply by LesFerch** [reply]
Hi Paddy. Did see the readme for my [SwitchExplorer](https://lesferch.github.io/SwitchExplorer/) and [OldExplorer](https://lesferch.github.io/OldExplorer/)? Did you see the options for doing direct property editing?

**Comment by Ulfenknulfen** [comment]
Hi LesFerch.

I've used WinSetView to modify my win 11 Explorer, and i use Oldexplorer to work in Folders i've need the Edit Option :-)

thanks for your Support
Paddy
